Sync/StorageClient.swift

```diff
@@ -645,7 +645,7 @@ open class Sync15CollectionClient<T: CleartextPayloadJSON> {
 
     // Exposed so we can batch by size.
     open func serializeRecord(_ record: Record<T>) -> String? {
-        return self.encrypter.serializer(record)?.rawString()
+        return self.encrypter.serializer(record)?.rawString(.utf8, options: [])
```

Comment:
Does the SwiftyJSON change also alter the handling of `nil` ➡️ `null` in the output? I know they make that an option, but I'm not sure what the old behavior was, or if we rely on it.

You should also make this change inside `KeyBundle.serializer(…)` (in two places). You should consider also making this change in `State.swift` — at least the `.utf8` part.

Here in `StorageClient.swift`, I think you also need to change line 464:

```swift
    func requestPOST(_ url: URL, body: [JSON], ifUnmodifiedSince: Timestamp?) -> Request {
        return self.requestPOST(url, body: body.map { $0.rawString()! }, ifUnmodifiedSince: ifUnmodifiedSince)
    }
```

Finally, there are lots of uses of `rawString` in `SyncTests/` that need to be adjusted.